classes/02class/exercises/c02-network09/marcio-anatrielo/c02-network09/c02-network09.txt

```diff
@@ -0,0 +1,29 @@
+
+
+# Answer for execise c02-network09 
+
+Description: SSH access from home to AWS
+
+Type: SSH
+Protocol: TCP
+Port Range: 22
+Source: 149.167.10.156/32
+
+When I create this rule to provide access from home to AWS becouse SG is state full both way will be allow to access. 
+
+
+
+Description: SSH access only from public instances on AWS
+
+Type: SSH
+Protocol: TCP
+Port Range: 22
+Source: 10.0.11.0/24, 10.0.13.0/24, 10.0.11.0/24
```

Comment:
the second  exercise you need to add the public SG as a source, not the IP ranges

Reply:
Cool I will change this. But on question in that case it is base on object not in IP address or Subnet?